docs/tools/cli/service/connection-info.rst

```diff
@@ -0,0 +1,211 @@
+
+.. code:: text
+
+  host='demo-pg-dev-project.aivencloud.com' port='13039' user=avnadmin dbname='defaultdb'
+
+
+``avn service connection-info pg uri``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Retrieves the connection URI for a certain Aiven for PostgreSQL® service.
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The name of the service
+  * - ``--route``
+    - The type of route to use to connect to the service. Possible values are ``dynamic``, ``privatelink`` and ``public``
+  * - ``--usage``
+    - The database connection usage. Possible values are ``primary`` and ``replica`` 
+  * - ``--privatelink-connection-id``
+    - The Id of the privatelink to use
+  * - ``--username``
+    - The username used to connect if using ``sasl`` authentication method
+  * - ``--dbname``
+    - The database name to use to connect
+  * - ``--sslmode``
+    - The ``sslmode`` to use. Possible values are ``require``, ``verify-ca``, ``verify-full``, ``disable``, ``allow``, ``prefer``
+
+
+**Example:** Retrieve the connection URI for an Aiven for PostgreSQL® service named ``demo-pg``:
+
+::
+
+  avn service connection-info pg uri demo-pg
+
+An example of ``avn service connection-info pg uri`` output:
+
+.. code:: text
+
+  postgres://avnadmin:XXXXXXXXXX@demo-pg-dev-project.aivencloud.com:13039/defaultdb?sslmode=require
+
+``avn service connection-info psql``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Retrieves the ``psql`` command needed to connect to a certain Aiven for PostgreSQL® service.
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The name of the service
+  * - ``--route``
+    - The type of route to use to connect to the service. Possible values are ``dynamic``, ``privatelink`` and ``public``
+  * - ``--usage``
+    - The database connection usage. Possible values are ``primary`` and ``replica`` 
+  * - ``--privatelink-connection-id``
+    - The Id of the privatelink to use
+  * - ``--username``
+    - The username used to connect if using ``sasl`` authentication method
+  * - ``--dbname``
+    - The database name to use to connect
+  * - ``--sslmode``
+    - The ``sslmode`` to use. Possible values are ``require``, ``verify-ca``, ``verify-full``, ``disable``, ``allow``, ``prefer``
+
+
+**Example:** Retrieve the ``psql`` command needed to connect to an Aiven for PostgreSQL® service named ``demo-pg``:
+
+::
+
+  avn service connection-info psql demo-pg
+
+An example of ``avn service connection-info psql`` output:
+
+.. code:: text
+
+  psql postgres://avnadmin:XXXXXXXXXXXX@demo-pg-dev-advocates.aivencloud.com:13039/defaultdb?sslmode=require
+
+
+``avn service connection-info redis uri``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Retrieves the connection URI needed to connect to a certain Aiven for Redis®* service.
```

Comment:
I don't think "certain" works here, you can just remove it and "connect to an Aiven for Redis service"